src/components/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import styles from "./SearchBar.module.scss";
+
+const SearchBar = ({ searchBooks, setSearchBooks }) => {
+    // const navigate = useNavigate();
+    const [newString, setNewString] = useState("");
+
+    const handleChange = (event) => {
+        setNewString(event.target.value);
+    };
```

Comment:
newString variable name could be more descriptive. something like searchQuery gives more context